ledger/participant-integration-api/src/main/scala/platform/index/ReadOnlySqlLedger.scala

```diff
@@ -204,13 +178,10 @@ private final class ReadOnlySqlLedger(
   override def currentHealth(): HealthStatus = ledgerDao.currentHealth()
 
   override def close(): Unit = {
-    // Terminate the dispatcher first so that it doesn't trigger new queries.
-    dispatcher.close()
```

Comment:
Removed this from here since its lifecycle should be handled by the provider.

Reply:
But is it?

Reply:
It is. These are constructed with the `Dispatcher.owner` which ensures releasing.